src/d_deh.c

```diff
@@ -1574,22 +1596,25 @@ void ProcessDehFile(const char *filename, char *outfilename, int lumpnum)
       dehfiles[i++] = strdup(filename);
       dehfiles[i] = NULL;
 
-      infile.lump = NULL;
+      infile.stream = NULL;
     }
   else  // DEH file comes from lump indicated by third argument
     {
-      infile.size = W_LumpLength(lumpnum);
-      infile.inp = infile.lump = W_CacheLumpNum(lumpnum, PU_STATIC);
-      filename = W_WadNameForLump(lumpnum);
+      void *buf = W_CacheLumpNum(lumpnum, PU_STATIC);
       // [FG] skip empty DEHACKED lumps
-      if (!infile.inp)
+      if (!buf)
         {
           printf("skipping empty DEHACKED lump from file %s\n", filename);
           return;
         }
+
+      infile.stream = mem_fopen_read(buf, W_LumpLength(lumpnum));
+      filename = W_WadNameForLump(lumpnum);
+
+      infile.file = NULL;
     }
 
-  printf("Loading DEH %sfile %s\n", infile.lump ? "lump from " : "", filename);
+  printf("Loading DEH %sfile %s\n", infile.file ? "lump from " : "", filename);
```

Comment:
This seems wrong, "lump from file" should report a PWAD.

Reply:
Wait, I think I misread the diff.

Reply:
I think you are right, I fixed it and renamed `stream` to `lump`: https://github.com/fabiangreffrath/woof/pull/1031/commits/caf08bc71ae6b92b4b4fa0cdf8467947e5c807c3

Reply:
Thanks, it was a bit confusing with the rename. 👍